### DISCUSSION TITLE
Email Verification to old Email

### DISCUSSION THREAD

**Discussion by gvillemos** [community/community#46414] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have an account that I havent used for around 15 years.

Trying to login, even after having provided the right password, I get a verification code send to the emaill address that no longer exist. I cant see any other ways of verifying myself.

![image](https://user-images.githubusercontent.com/95567969/216811489-a9f01ce6-9b95-4241-a4b7-7c20692f5d7b.png)

How can I recover my old account?

**Comment by jge162** [comment, marked as answer]
100% you need to reach out to support, we can only discuss it here but they cant possibly help you recover access. But, you have to speak with them. Here is the link to [Support](https://support.github.com/request/account?tags=hubberfy_account)

choose the topic sign in issues

**Reply by jge162** [reply]
<img width="1063" alt="Screenshot 2023-02-05 at 1 40 08 AM" src="https://user-images.githubusercontent.com/31228460/216811844-7c54ebc5-9e19-4530-bff0-6193b731b209.png">
